.ladle/adapters/PlainComponentAdapter.tsx

```diff
@@ -878,4 +889,72 @@ export const PlainComponentAdapter: ComponentsContextType = {
       </div>
     )
   },
+
+  Table: <T,>({
```

Comment:
Hmm so i'm a little worried about consumers needing to support generics 🤔 i'm wondering if we could simplify. let's chat about it tomorrow. Not a blocker, just something to think about with other components

My thought here is that ultimately the consumer is only concerned about what to render. So i'm wondering if there's a version of the Table adapter we can expose that will effectively provide them with data with react nodes instead of needing them to call render functions. Some of the work for generics is for our flexibility on the SDK side of things to work with data sets, but the consumer just needs to know about what to render

Reply:
Yeah I think the reason I went with this approach was I wanted the column definitions to be able to be strongly typed. I'm willing to revisit this if you think there is a better approach.